src/basics.js

```diff
@@ -4,13 +4,13 @@ import * as base32 from './bases/base32.js'
 import * as base58 from './bases/base58.js'
 import * as sha2 from './hashes/sha2.js'
 
-import raw from './codecs/raw.js'
-import json from './codecs/json.js'
+import { rawCodec } from './codecs/raw.js'
+import { jsonCodec } from './codecs/json.js'
```

Comment:
I would just exported those as `raw` and `json` especially since they are renamed to that later.